apps/demos/Demos/Chat/Overview/Angular/app/app.component.css

```diff
@@ -0,0 +1,20 @@
+.demo-container {
+    height: 100%;
+    padding: 30px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 30px;
```

Comment:
According to demo-prototype the gap between Chats is 20px.

Reply:
57acc8aa4c51d5f8d5e5517fb13187024a0be1cd